### DISCUSSION TITLE
Clipboard Issue

### DISCUSSION THREAD

**Discussion by robertschindler** [mRemoteNG/mRemoteNG#2555] [Q&A]
Hello to all

I hope somebody can help me...

Since I use a new notebook, I have some troubles working with mRemote...

environment:
- win 11 (22H2 22621.3007)
- mRemote 1.77.3 (NB 1784) - also tested with older versions

In case I start connection to a remote server and there is "redirect clipboard" acitvated, then the whole clipboard (even an my local maschine) is fully disabled...

so I am not able to do a copy/paste neither between remote/local nor between applications on the local PC.

do I have to configure something inside of Win 11 or do I have to do a special config of mRemote?!?

many thanks for your answers!
best regards Robert

**Comment by Kvarkas** [comment]
I use it on win11 and hasn't spotted, what if you not enable redirect clipboard? Is that effect all connections you are making?
